packages/components/src/components/contract-card/result-overlay/__tests__/result-overlay.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ResultOverlay from '../result-overlay';
+
+const closed = 'Closed';
+const default_mocked_props = {
+    currency: 'USD',
+    contract_id: 1987232323,
+    getCardLabels: () => {
+        return {
+            CLOSED: closed,
+        };
+    },
+    getContractPath: jest.fn(),
+    is_positions: true,
+    is_unsupported: false,
+    is_visible: true,
+    onClick: jest.fn(),
+    onClickRemove: jest.fn(),
+    payout_info: 4.05,
+    result: 'won',
+};
+
+jest.mock('react-transition-group', () => ({
+    CSSTransition: jest.fn(({ children }) => <div>{children}</div>),
+}));
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    NavLink: jest.fn(({ children }) => <div>{children}</div>),
+}));
+
+describe('<ResultOverlay/>', () => {
+    it('should render component with plus inside if payout_info is positive', () => {
+        render(<ResultOverlay {...default_mocked_props} />);
+
+        expect(screen.getByText(closed)).toBeInTheDocument();
```

Comment:
@kate-deriv nit-pick: we can get rid of closed variables by directly using getCardLabels().CLOSED :) 
```suggestion
        expect(screen.getByText(default_mocked_props.getCardLabels().CLOSED)).toBeInTheDocument();
```
and we can have a shorthand at line 9 above:
```
    getCardLabels: () => ({ CLOSED: 'Closed' }),
```

Reply:
Nice catch) 
Will apply

Reply:
I have an unhealthy addiction to turning all strings into constants))))

Reply:
thank you ) btw, what about the shorthand?  `getCardLabels: () => ({ CLOSED: 'Closed' })` :)